packages/zk-transfer-web/src/libs/api.ts

```diff
@@ -31,24 +38,23 @@ export const makeApi = async (demo: any): Promise<apiObject> => {
   const demoAztec = window.aztec;
   const chainId = await eth.net_version();
   const bm = await demo.contract.createBM({ chainId, autoConfig: true });
-  const deploys: TokenList<Token> = await bm.getDeploys();
+  const deploys: TokenMap = await bm.getDeploys();
```

Comment:
A better name for this type is `TokenDeploys` here and everywhere else :)
please change

Reply:
Done!